sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -3837,6 +4229,50 @@ public partial class SenderAuthorization : System.ClientModel.Primitives.IJsonMo
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.SenderAuthorization>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.SenderAuthorization>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    public partial class Service : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.Service>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Service>
```

Comment:
Single word name is not allowed, could we add some prefix to specify it?

Reply:
Renamed to `PipelineGroupService`